test/storage/memory.test.ts

```diff
@@ -56,8 +56,13 @@ describe('tests memory storage', () => {
     data.data = 'another data';
 
     expect(storage.data['key']).not.toBeNull();
-    expect(storage.data['key'].state).toBe('cached');
-    expect(storage.data['key'].data.data).toBe('data');
+    if (storage.data['key']) {
+      expect(storage.data['key'].state).toBe('cached');
```

Comment:
```suggestion
      expect(storage.data['key']!.state).toBe('cached');
```

Use the `!` operator and remove these if checks.